lib/destiny-2-api.js

```diff
@@ -44,6 +44,38 @@ class Destiny2API {
         return promiseRequest(this.options).then(res => formatJson(res));
     }
 
+    /**
+    * Gets information of a Bungie.net Account
+    * @param {number} id The requested Bungie.net membership id.
+    */
+    getBungieNetUserById(id) {
+        this.options.path = `/Platform/User/GetBungieNetUserById/${id}/`;
+        this.options.method = 'GET';
+        return promiseRequest(this.options).then(res => formatJson(res));
+    }
+
+    /**
+    * Gets Destiny memberships given a bungie name
+    * @param {string} displayName bungie name ex: Hello#1234
+    * @param {number} membershipType enum of values for specifying platform
+    */
+    SearchDestinyPlayerByBungieName(displayName, displayNameCode) {
+        this.options.path = '/Platform/Destiny2/SearchDestinyPlayerByBungieName/All/'
+        this.options.method = 'POST';
+        return promisePost(this.options, {displayName: displayName, displayNameCode: displayNameCode}).then(res => formatJson(res));
```

Comment:
nit: one more spacing suggestion, since the line is so long
```suggestion
        return promisePost(this.options, { displayName: displayName, displayNameCode: displayNameCode })
            .then(res => formatJson(res));
```